qiita_db/support_files/populate_test_db.sql

```diff
@@ -488,7 +488,7 @@ INSERT INTO qiita.message_user (message_id, email) VALUES (1, 'test@foo.bar'),(1
 
 -- Create a loggin entry
 INSERT INTO qiita.logging (time, severity_id, msg, information)
-    VALUES ('Sun Nov 22 21:29:30 2015', 2, 'Error message', NULL);
+    VALUES ('Sun Nov 22 21:29:30 2015', 2, 'Error message', '{}');
```

Comment:
This shouldn't be blocking so I added an issue: https://github.com/biocore/qiita/issues/1553